src/runtime/operatormethod.cs

```diff
@@ -49,6 +49,13 @@ static OperatorMethod()
                 ["op_OnesComplement"] = new SlotDefinition("__invert__", TypeOffset.nb_invert),
                 ["op_UnaryNegation"] = new SlotDefinition("__neg__", TypeOffset.nb_negative),
                 ["op_UnaryPlus"] = new SlotDefinition("__pos__", TypeOffset.nb_positive),
+                ["op_OneComplement"] = new SlotDefinition("__invert__", TypeOffset.nb_invert),
+                ["op_Equality"] = new SlotDefinition("__eq__", TypeOffset.tp_richcompare),
+                ["op_Inequality"] = new SlotDefinition("__ne__", TypeOffset.tp_richcompare),
+                ["op_LessThanOrEqual"] = new SlotDefinition("__le__", TypeOffset.tp_richcompare),
+                ["op_GreaterThanOrEqual"] = new SlotDefinition("__ge__", TypeOffset.tp_richcompare),
+                ["op_LessThan"] = new SlotDefinition("__lt__", TypeOffset.tp_richcompare),
+                ["op_GreaterThan"] = new SlotDefinition("__gt__", TypeOffset.tp_richcompare),
```

Comment:
I think this is error-prone. `op_LessThan` does not really implement `tp_richcompare`. It would be better to have a separate `ComparisonOpMap`, `string -> string`, and fix `IsOperatorMethod` correspondingly.

It would also remove the need in the change in `FixupSlots` below.

Reply:
Makes sense, thanks! Should that `ComparisonOpMap` also be merged with the other map for `op_LessThan` -> `Py_LT`?

Reply:
I added this `OperatorMethod.ComparisonOpMap` in the latest commit, without merging it with the `ClassBase.CilToPyOpMap`, although they both have the same keys so it would make sense to put them together in `OperatorMethod`.